R/coord-cartesian-.r

```diff
@@ -144,24 +144,37 @@ view_scales_from_scale <- function(scale, coord_limits = NULL, expand = TRUE) {
   view_scales
 }
 
-panel_guide_label <- function(guides, position, default_label) {
-  guide <- guide_for_position(guides, position) %||% guide_none(title = waiver())
-  guide$title %|W|% default_label
-}
-
 panel_guides_grob <- function(guides, position, theme) {
-  guide <- guide_for_position(guides, position) %||% guide_none()
-  guide_gengrob(guide, theme)
+  guide <- guide_for_position(guides, position) %||% list(guide = guide_none())
+  guide$guide$draw(theme, guide$params)
```

Comment:
They are, but perhaps variable names were chosen confusingly.
In L148, `guide_for_position()` gives a named list with a guide object - parameter pair.
Hence, `guide$params` is not accessing a guide object's slots but just a list element named `params`.
Will rename to avoid confusion.